### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by LenkaSeg**

### CODE REVIEW

File: tests/test_integrate.py [hunk 1]
```diff
@@ -182,3 +182,14 @@ def test_log(loglines):
...
+        # Test with csv_dump option
+        csv_dump = open(f"{tmp_dir}/test.csv", "w+")
+        totals(
+            countme_totals=totalsdb,
+            countme_raw=rawdb,
+            progress=False,
+            csv_dump=csv_dump,
+        )
+        csv_dump.seek(0)
+        assert len(csv_dump.readlines()) > 0
```
Comment (nphilipp) [inline tests/test_integrate.py]: The second call to `totals()` also writes the sqlite DB, so both can be rolled into one:

```
        csv_dump = open(f"{tmp_dir}/test.csv", "w+")
        totals(
            countme_totals=totalsdb,
            countme_raw=rawdb,
            progress=False,
            csv_dump=csv_dump,
        )
        db = sqlite3.connect(totalsdb)
        rows_no = db.execute("select count(*) from countme_totals;").fetchone()[0]
        assert int(rows_no) > 0
        csv_dump.seek(0)
        assert len(csv_dump.readlines()) > 0
```

I'll commit this into your branch.

**Review by nphilipp — CHANGES_REQUESTED** [review]
Please review my suggestion and if applicable, squash the commits and merge. Thanks!

**Comment by LenkaSeg** [comment]
Done, thanks!

**Review by nphilipp — APPROVED** [review]
Please review my suggestion and if applicable, squash the commits and merge. Thanks!

**Pull request merged**